### DISCUSSION TITLE
How to build [termux-packages] repo using GitHub actions

### DISCUSSION THREAD

**Discussion by rtx2080tix** [termux/termux-app#2700] [Q&A] [answered]
My pc has 
512 GB hard disk 
4 GB ram 
dual core CPU
 I am posting using this account because I can't access my other account(i lost my 2fa on my other account cpuuntery)
I edited [properties.sh](https://github.com/rtx2080tix/termux-packages/blob/master/scripts/properties.sh) but i am unable to build the bootstrap or any package for that matter

**Comment by sylirre** [comment, marked as answer]
See these links for building package locally:

* https://github.com/termux/termux-packages/wiki/Build-environment
* https://github.com/termux/termux-packages/wiki/Building-packages

However for using GitHub Actions you don't need a PC. Just fork the repo, enable GitHub Actions in repo settings. Then you should be able to either build packages automatically by pushing changes to respective `build.sh` file or manually by using workflow dispatch trigger:

![gha](https://user-images.githubusercontent.com/107305601/232223340-93c60a63-160c-467f-a5a6-d66ef7e37883.png)
